demo/admin/src/news/generated/NewsGrid.tsx

```diff
@@ -114,6 +115,16 @@ export function NewsGrid(): React.ReactElement {
         },
         { field: "slug", headerName: intl.formatMessage({ id: "news.slug", defaultMessage: "Slug" }), width: 150 },
         { field: "title", headerName: intl.formatMessage({ id: "news.title", defaultMessage: "Title" }), width: 150 },
+        {
+            field: "status",
+            headerName: intl.formatMessage({ id: "news.status", defaultMessage: "Status" }),
+            type: "singleSelect",
+            valueOptions: [
+                { value: "Active", label: intl.formatMessage({ id: "news.status.active", defaultMessage: "Active" }) },
+                { value: "Deleted", label: intl.formatMessage({ id: "news.status.deleted", defaultMessage: "Deleted" }) },
```

Comment:
There shouldn't be a filter for deleted and active, right? We treat delete as a soft delete as far as I remember, meaning, the deleted items should be shown in a separate dialog.

Reply:
Yes, but admin generator doesn't support the new Status yet